Kickstarter-iOS/Features/RewardsCollection/Controller/RewardsCollectionViewControllerTests.swift

```diff
@@ -77,24 +74,23 @@ final class RewardsCollectionViewControllerTests: TestCase {
           |> Backing.lens.rewardId .~ reward.id
           |> Backing.lens.shippingAmount .~ 10
           |> Backing.lens.amount .~ 700.0
+          |> Backing.lens.addOns .~ []
```

Comment:
Opting out of add ons here ensures that this reward will be shown as unavailable when it's not selected. The default addOn has the same reward id as one of the rewards, which messes with our count for the reward.